### DISCUSSION TITLE
Fusion360 crashes on startup - How to debug this? *solved*

### DISCUSSION THREAD

**Discussion by kekskekser** [cryinkfly/Autodesk-Fusion-360-for-Linux#140] [Q&A] [answered]
Hello,

I stumbled across this repository again after reading a news article about the wine 6.22 release which supposedly fixes some issues when using fusion360. So I decided to give this a try again and see if I can run fusion360 on my system.

First off my build specs:
AMD 200GE (using the iGPU)
8GB RAM
Ubuntu 20.04
Wine 6.21 staging, later 6.22 development branch

So the installation presumably went fine and I followed the instructions provided in the readme to the letter.
Upon first starting fusion360, I was greeted by a prompt to create an autodesk account. I obliged and signed up and in.

After waiting for about a minute in front of the splash screen a "Fusion 360 Error Report" window popped up.
And that is as far as I have gotten.

I soon realized that I was only using wine 6.21 staging, so I uninstalled/purged everything and started the process from the ground up, using wine 6.22 from the development branch. However, this didn't change the resulting behavior, and so I figured I might redo the process again, but this time using dxvk instead of the recommended opengl. No difference.

I'm not exactly sure how to continue from here on. I would greatly appreciate any assistance on how to pursue this or debug this.
The error report window does reference some files, and given that they don't include any private data, I would love to share those publicly if it helps.

Thank you for this project and your continued efforts to make fusion360 usable on linux.

**Comment by tome9111991** [comment, marked as answer]
Maybe you use wrong launcher?
You try start fusion360 with desktop shortcut?

**Reply by kekskekser** [reply]
Hi,

Thank you for your quick response!
Yes, I did use the desktop shortcut to launch fusion360.

**Reply by tome9111991** [reply]
The desktop shortcut dosent work.

Go try to find fusion360 starter in the "Start menu"
For expemple on linux mint i can finde fusion360 in wine category in start menu.

**Reply by cryinkfly** [reply]
For example (old path): 

![Bildschirmfoto_2021-11-24_12-28-58](https://user-images.githubusercontent.com/79079633/143230225-b6291b3e-48be-403f-832b-8342470c8ae5.png)

The new path for the **standard installation** is:

> $HOME/.local/applications/wine/Programs/Autodesk/fusion360-launcher.sh

**Reply by kekskekser** [reply]
Hello,

Launching the script myself, or using the entry in the start menu, leads to fusion booting up, but the display turning black, and the computer rebooting 30 seconds after that. I did try using different wine versions again as well as switching between opengl and dxvk, but again this did not change the behavior.
That said, I assume that this issue has nothing to do with this project and is either a driver-/wine/- or system-related issue, or my hardware is simply incompatible.
I will try to get this to work every couple of weeks and see if the situation changes for me.
But so far thank you very much for your efforts and your help.

**Reply by cryinkfly** [reply]
@kekskekser How you install Autodesk Fusion 360? Which version of my setup wizard did you have used?
